ddtrace/opentracer/scope.py

```diff
@@ -1,10 +1,20 @@
-from opentracing import Scope as OpenTracingScope
+import opentracing
 
 
-class Scope(OpenTracingScope):
+class Scope(opentracing.Scope):
     """"""
 
+    __slots__ = ['_manager', '_span']
```

Comment:
[`opentracing.Scope`](https://github.com/opentracing/opentracing-python/blob/2.0.0rc2/opentracing/scope.py#L24) doesn't use `__slots__` so there's nothing to gain by using it here (see [notes](https://docs.python.org/2.7/reference/datamodel.html#slots))